README.md

```diff
@@ -0,0 +1,60 @@
+# Mountsion CBS Secondary Website
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Mount Sion Crest.jpg)
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford. As time goes on more students will resume the work that we havn't finished and will continue to keep the website up to date.
```

Comment:
I think we could maybe reword this again if you don't mind :) `Mount Sion C.B.S school website, built and maintained by transition year students. This project teaches students the fundamentals of web design and development along with project management following the agile methodology. `